src/Symfony/Bridge/Twig/Extension/HttpKernelExtension.php

```diff
@@ -28,7 +28,7 @@ public function getFunctions(): array
             new TwigFunction('render', [HttpKernelRuntime::class, 'renderFragment'], ['is_safe' => ['html']]),
             new TwigFunction('render_*', [HttpKernelRuntime::class, 'renderFragmentStrategy'], ['is_safe' => ['html']]),
             new TwigFunction('fragment_uri', [HttpKernelRuntime::class, 'generateFragmentUri']),
-            new TwigFunction('controller', static::class.'::controller'),
+            new TwigFunction('controller', static::controller(...)),
```

Comment:
class is final, it'd make sense to keep the style of previous lines actually (it makes twig take an easier path when optimizing the call):
```suggestion
            new TwigFunction('controller', [self::class, 'controller']),
```

Reply:
okay 👍🏼 fixed, thanks